pkg/descheduler/strategies/pod_antiaffinity_test.go

```diff
@@ -149,12 +156,26 @@ func TestPodAntiAffinity(t *testing.T) {
 			nodes:                   []*v1.Node{node1, node3},
 			expectedEvictedPodCount: 0,
 			nodeFit:                 true,
+			testFunc:                func(podList []v1.Pod) {},
+		},
+		{
+			description:             "No pod to evicted since all pod terminating",
+			maxPodsToEvictPerNode:   0,
+			pods:                    []v1.Pod{*p1, *p2, *p3, *p4},
+			nodes:                   []*v1.Node{node1},
+			expectedEvictedPodCount: 3,
+			testFunc: func(podList []v1.Pod) {
```

Comment:
Why 4 pods? Are not two pods sufficient enough?

Reply:
Hi . This is for comare with above test case.
```
			description:             "Maximum pods to evict - 3",
			maxPodsToEvictPerNode:   3,
			pods:                    []v1.Pod{*p1, *p2, *p3, *p4},
			nodes:                   []*v1.Node{node1},
			expectedEvictedPodCount: 3,
```

Reply:
Fix done~ Please check :)